ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthPeers.java

```diff
@@ -197,8 +195,11 @@ private boolean registerDisconnect(
         disconnectCallbacks.forEach(callback -> callback.onDisconnect(peer));
         peer.handleDisconnect();
         abortPendingRequestsAssignedToDisconnectedPeers();
-        LOG.debug("Disconnected EthPeer {}", peer.getShortNodeId());
-        LOG.trace("Disconnected EthPeer {}", peer);
+        if (peer.getReputation().getScore() > 102) {
```

Comment:
Can you have this in a field? maybe something like: `USEFULL_PEER_SCORE_THRESHOLD`

Reply:
Nice!